File: AndroidStudioProjects/UberClone/app/src/main/java/com/example/uberclone/MainActivity.java
package com.example.uberclone;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.Switch;

import com.parse.LogInCallback;
import com.parse.ParseAnonymousUtils;
import com.parse.ParseException;
import com.parse.ParseUser;

public class MainActivity extends AppCompatActivity {
    boolean isDriverModeActive;
    public void getStarted(View v) {
        if (isDriverModeActive) {

        } else {

        }
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        getSupportActionBar().hide();

        Switch RiderDriverSwitch = findViewById(R.id.switch1);
        isDriverModeActive = RiderDriverSwitch.isChecked();
    }
}

File: AndroidStudioProjects/App1/app/src/main/java/com/example/app1/MainActivity.kt
package com.example.app1

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.databinding.DataBindingUtil
import com.example.app1.databinding.ActivityMainBinding


class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val binding: ActivityMainBinding =
            DataBindingUtil.setContentView(this, R.layout.activity_main)
    }
}
